invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/ActionDropdown.js

```diff
@@ -84,7 +84,7 @@ class ActionDropdown extends Component {
             {actionSuccess && (
               <SuccessIcon timeOutDelay={3000} show={actionSuccess} />
             )}
-            {error && <ErrorPopup error={error} />}
+            {!actionSuccess && error && <ErrorPopup error={error} />}
```

Comment:
```suggestion
            {error && <ErrorPopup error={error} />}
```
since you are clearing the error in line 48, this additional check is not needed

Reply:
Totally right :)